2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -24,6 +24,22 @@ collector {
   port = {{collectorPort}}
   port = ${?COLLECTOR_PORT}
 
+  # The collector responds with a cookie to requests with a path that matches the 'vendor/version' protocol.
+  # The expected values are:
+  # - com.snowplowanalytics.snowplow/tp2 for Tracker Protocol 2
+  # - r/tp2 for redirects
+  # - com.snowplowanalytics.iglu/v1 for the Iglu Webhook
+  # Any path that matches the 'vendor/version' protocol will result in a cookie response, for use by custom webhooks
+  # downstream of the collector.
+  # But you can optionally also map any valid (i.e. two-segment) path to one of the three defaults.
+  # Your custom path must be the key and the value must be one of the corresponding default paths.
+  # Uncomment to use. Defaults to empty map.
+  #paths {
+  #  "com.acme/track" = "com.snowplowanalytics.snowplow/tp2"
```

Comment:
I tend to agree with Anton, we're better off with fully qualified paths

Reply:
I've changed it to the way you guys advise.